api_pagopa.yaml

```diff
@@ -0,0 +1,264 @@
+  description: Proxy for PagoPA payment services
+basePath: /api/v1
+schemes:
+  - https
+paths:
+  '/payment-requests/{rptId}':
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: getPaymentInfo
+      summary: Get Payment Info
+      description: Retrieve information about a payment
+      parameters:
+        - $ref: '#/parameters/RptIdFromString'     
+      responses:
+        '200':
+          description: Payment information retrieved
+          schema:
+            $ref: '#/definitions/PaymentRequestsGetResponse'
+          examples:
+            application/json:
+              importoSingoloVersamento: 200,
+              codiceContestoPagamento: 'ABC123'
+        '400':
+          description: Invalid input or PagoPA response
+        '500':
+          description: PagoPA services are not available or request is rejected
+  '/payment-activations':
+    x-swagger-router-controller: PaymentController
+    post:
+      operationId: activatePayment
+      summary: Activate Payment
+      description: Require a lock (activation) for a payment
+      parameters:
+        - in: body
+          name: body
+          schema:
+            $ref: '#/definitions/PaymentActivationsPostRequest'
+          x-examples:
+            application/json:
+              rptId: '12345678901012123456789012345'
+              importoSingoloVersamento: 200
+              codiceContestoPagamento: 'ABC123'
+      responses:
+        '200':
+          description: Payment activation process started
+          schema:
+            $ref: '#/definitions/PaymentActivationsPostResponse'
+          examples:
+            application/json:
+              importoSingoloVersamento: 200
+        '400':
+          description: Invalid input or PagoPA response
+        '500':
+          description: PagoPA services are not available or request is rejected
+  '/payment-activations/{codiceContestoPagamento}':
+    x-swagger-router-controller: PaymentController
+    get:
+      operationId: getActivationStatus
+      summary: Get the activation status
+      description: Get the activation status and the paymentId
+      parameters:
+        - $ref: '#/parameters/CodiceContestoPagamento' 
+      responses:
+        '200':
+          description: Activation information
+          schema:
+            $ref: '#/definitions/PaymentActivationsGetResponse'
+          examples:
+            application/json:
+              idPagamento: '123455'
+        '400':
+          description: Invalid input
+        '404':
+          description: Activation status not found
+        '500':
+          description: Service unavailable
+definitions:
+  CodiceContestoPagamento:
+    description: Transaction Id used to identify the communication flow
+    type: string
+    minLength: 1
+    maxLength: 35
+  EnteBeneficiario:
+    description: Beneficiary institution related to a payment
+    type: object
+    properties:
+      identificativoUnivocoBeneficiario:
```

Comment:
isn't this the fiscal code of the ente? - cc @gunzip

Reply:
it's not clear from the pagopa documentation if it is *only* the fiscal code (I guess, it might be):
"Aggregazione che riporta le informazioni concernenti l’identificazione fiscale dell’ente beneficiario."